### PR TITLE
DCKUBE-664: Added a recommendation to report product bugs in Atlassian Support page

### PR DESCRIPTION
**Pull request by nghazali**
Recommended to report product bugs in Atlassian Support page instead of GitHub itself

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/report-issue.md [hunk 1]
```diff
@@ -1,34 +1,42 @@
 ---
-name: Report Issue
-about: Report an issue to help us improve
+name: Report A Helm Chart Issue
+about: Report helm chart issues to help us improve
```
Comment (errcode1202) [inline .github/ISSUE_TEMPLATE/report-issue.md:3]: @tan-ro should `helm chart` be uppercased - `Helm Chart`?

File: .github/ISSUE_TEMPLATE/report-issue.md [hunk 2]
```diff
@@ -1,34 +1,42 @@
...
 title: "[ISSUE]"
 labels: bug
 assignees: ''
 
 ---
 
+### NOTE:
+Please report only helm chart issues here. To report a product specific bug use [Attlassian Support Page](https://jira.atlassian.com/secure/Dashboard.jspa)
```
Comment (errcode1202) [inline .github/ISSUE_TEMPLATE/report-issue.md:11]: Same here  - `Helm Chart`?

Comment (errcode1202) [inline .github/ISSUE_TEMPLATE/report-issue.md:11]: Typo in hyperlink - `Attlassian` you have two `t`

File: .github/ISSUE_TEMPLATE/report-issue.md (hunk 1 above)
Comment (errcode1202) [inline .github/ISSUE_TEMPLATE/report-issue.md:3]: Not sure about this sentence, `Report helm chart issues to help us improve`, it feels like its hanging and not complete. @tan-ro any recommendations?

File: .github/ISSUE_TEMPLATE/report-issue.md [hunk 3]
```diff
@@ -1,34 +1,42 @@
...
+
+### Report the bug:
```
Comment (errcode1202) [inline .github/ISSUE_TEMPLATE/report-issue.md:13]: Should it be `bug` or `issue`? For consistency maybe `issue`?

**Review by errcode1202 — CHANGES_REQUESTED** [review]
Some typos

**Review by tan-ro — APPROVED** [review]

**Review by tarka — APPROVED** [review]

**Review by errcode1202 — APPROVED** [review]

**Pull request merged**